packages/identity-service/src/routes/authentication.js

```diff
@@ -44,26 +130,6 @@ module.exports = function (app) {
             paranoid: false
           })
           if (!existingRecord) {
-            // default to null
-            let walletAddress = null
```

Comment:
moved to top of function so recovered walletAddress is accessible in multiple spots